python/build.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# Generates Python protocol buffers and adjusts package namespaces
```

Comment:
We are packaging the proto messages under the `matrix_io.proto` namespace, this requires some minor tweaking.